FoodOrderWebApi/Configuration/DbInitializer.cs

```diff
@@ -0,0 +1,215 @@
+﻿using FoodOrderWebApi.Model;
+using NodaTime;
+
+namespace FoodOrderWebApi.Configuration
+{
+    public class DbInitializer
+    {
+        private static FoodOrderDbContext _context = null!;
+
+        public static void Initialize(IServiceProvider serviceProvider)
+        {
+            _context = serviceProvider.GetRequiredService<FoodOrderDbContext>();
+
+            if (!_context.Database.EnsureCreated())
+            {
+                return;
+            }
+
+            SeedCategories();
+            SeedOpeningHours();
+            SeedRestaurants();
+            SeedProducts();
+        }
+        private static void SeedCategories()
+        {
+            var categories = new Category[]
+            {
+                new Category
+                {
+                    Name = "Soup",
+                    PictureName = "soup.jpg"
+                },
```

Comment:
I think it would condense the code a little bit if you made a factory function for this e.g. `makeCategory("Soup", "soup.jpg")`